src/hexdeck.gleam

```diff
@@ -132,6 +133,7 @@ fn update_query(order: List(Int), frames: Dict(Int, String)) -> Effect(msg) {
           list.filter_map(order, fn(key) {
             frames
             |> dict.get(key)
+            |> result.try(fn(param) { Ok(string.lowercase(param)) })
```

Comment:
we can write this as:

```gleam
|> result.map(string.lowercase)
```

instead, I think that's a bit clearer.

Reply:
Absolutely, thanks for the suggestion!